org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/tests/DefaultWollokTestsReporter.xtend

```diff
@@ -14,7 +14,7 @@ import wollok.lib.AssertionException
  */
 class DefaultWollokTestsReporter implements WollokTestsReporter {
 	
-	override reportTestAssertError(WTest test, AssertionException assertionError, int lineNumber, URI resource) {
+	override reportTestAssertError(WTest test, AssertionException assertionError, WollokObject wollokException, int lineNumber, URI resource) {
```

Comment:
Is it ok now to have both objects ? AssertionException AND the wollokObject for the exception ? Couldn't we just have one ?
This is called with the following values from the interpreter

```
new AssertionException(e.wollokMessage), e.wollokException,
```

See, AssertionException is created with the message of the wollok object.

I wonder if we could actually delete AssertionException.